python/tests/test_native_skills/FileIOSkill/native_function.py

```diff
@@ -0,0 +1,74 @@
+import os
```

Comment:
Looks like this is a copy of file_io_skill.py. 

To make this skill more unique and testable, maybe something like hello_world_skill would be appropriate? I'm not opposed to this though!

Reply:
Sure I'll change